src/libmongoc/tests/unified/util.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef UNIFIED_UTIL_H
+#define UNIFIED_UTIL_H
+
+#include "mongoc/mongoc.h"
+
+mongoc_write_concern_t * bson_to_write_concern (bson_t *bson, bson_error_t* error);
```

Comment:
I feel like this is a slightly different style from our normal header declarations?  Clang format?

Reply:
Ah I did forget to run clang-format. Done.